packages/navigator/src/browser/navigator-model.ts

```diff
@@ -97,6 +97,9 @@ export class FileNavigatorModel extends FileTreeModel {
      * @returns file tree node if the file with given uri was revealed, undefined otherwise
      */
     async revealFile(uri: URI): Promise<TreeNode | undefined> {
+        if (!uri.path.isAbsolute) {
+            return undefined;
```

Comment:
nice catch!

Reply:
took me couple hours to figure out, no way to see from the dev tool, the page just gets completely unresponsive 😬